nextflow.config

```diff
@@ -52,7 +52,11 @@ params {
     // Variant Calling
     only_paired_variant_calling = false //if true, skips germline variant calling for normal-paired samples
     ploidy = 2          //null (in ascat, test this works) // Use default value, you can use 2,3,4
-    ascat_purity = null // Use default value
+    ascat_chromosomes = "c(1:22, 'X')"  // default value for ASCAT
```

Comment:
my bad, overlooked this. However, what about when we have males, then it should also have 'Y'